pkgs/os-specific/darwin/osxsnarf/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, lib, fetchFromGitHub, plan9port, darwin, ... }:
+
+stdenv.mkDerivation rec {
+  pname = "osxsnarf";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "eraserhd";
+    repo = "osxsnarf";
+    rev = "v${version}";
+    sha256 = "1vpg39mpc5avnv1j0yfx0x2ncvv38slmm83zv6nmm7alfwfjr2ss";
+  };
+
+  buildInputs = [ plan9port darwin.apple_sdk.frameworks.Carbon ];
+
+  installPhase = ''
+    make prefix=$out install
+  '';
+
+  meta = with lib; {
+    description = "A Plan 9-inspired way to share your OS X clipboard";
+    homepage = https://github.com/eraserhd/osxsnarf;
+    license = licenses.unlicense;
```

Comment:
also, you should put a license :)

Reply:
This was a very old piece of code I resurrected.   I talked to the original author, and "unlicense" was close enough to the original intent, because "we didn't worry about licenses too much those days."  (There's an official-looking `UNLICENSE` in the source repo.)